bundle/src/main/java/com/adobe/acs/commons/analysis/jcrchecksum/impl/ServletConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2015 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package com.adobe.acs.commons.analysis.jcrchecksum.impl;
+
+public final class ServletConstants {
```

Comment:
This is a personal opinion: but I usually use interfaces instead of final classes when creating a constants holder. Default state is no constructor, and all attributes are public static final.